seatunnel-api/src/main/java/org/apache/seatunnel/api/table/type/SeaTunnelDataType.java

```diff
@@ -33,4 +33,9 @@
      * Gets the SQL standard type represented by this data type.
      */
     SqlType getSqlType();
+
+    default Integer getLength() {
```

Comment:
Hi, I can't find where use this method?

Reply:
Yes, I have remove this method, this method is used to define the column length, e.g varcher(256).
This method has been moved in `Column`, and I will remove this in SeaTunnelDataType.